### ISSUE TITLE
Debug tries to run nme instead of lime

### ISSUE THREAD

**Issue by ubald** [TiVo/intellij-haxe#154]
Clicking debug tries to launch with

```
/usr/bin/haxelib run nme run "" mac -debug -Ddebug -args -start_debugger -debugger_host=localhost:6972
```

instead of lime.

It also doesn't use the configured project.xml as seen by the empty double quotes.

TIR: STB-9362

**Comment by as3boyan** [comment]
Actually you have to manually setup it in Project Options. Perhaps we can detect if project is OpenFL or NME, or XML, or Haxe project... But still it's complicated, maybe we can just fill project info on start, if it's empty, for example put there some xml if it is related to OpenFL or Lime, maybe fill other project types with simple things, but still it's tricky, for example there is no project template for OpenFL and Lime, so, if you can suggest some logic, we might implement it, if it makes sense.

**Comment by ubald** [comment]
The project is already configured and compiling/launching correctly when I use "Run". Couldn't the "Debug" launch in the same way as "Run" but with -debug and stuff?

**Comment by as3boyan** [comment]
Sorry, then. So OpenFL project/Mac target runs `nme`

**Comment by ubald** [comment]
Yes, but only when debugging from intellij, not running. Sorry if I wasn't clear enough.

**Comment by EBatTiVo** [comment]
@sganapavarapu1 fixed this with change 0c992ef4bdb50bef640fc453311ed00d900fbdc4.  This was included in the 0.9.4 release.

@ubald Please test and re-open if the error recurs.
